src/include/gurt/telemetry_common.h

```diff
@@ -18,6 +18,43 @@
 #define D_TM_SHARED_MEMORY_KEY		0x10242048
 #define D_TM_SHARED_MEMORY_SIZE		(1024 * 1024)
 
+#define D_TM_MIN(stats, name) \
+	stats->dtm_min.min_##name
+
+#define D_TM_MAX(stats, name) \
+	stats->dtm_max.max_##name
+
+#define D_TM_PRINT_STATS(s, stats, name, fmt) \
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon